tools/pd-ctl/pdctl/ctl.go

```diff
@@ -198,3 +198,15 @@ func genCompleter(cmd *cobra.Command) []readline.PrefixCompleterInterface {
 	}
 	return pc
 }
+
+// ReadStdin convert stdin to string array
+func ReadStdin(r io.Reader) (input []string, err error) {
```

Comment:
Since the name of it is `ReadStdin`, I think there is no need to pass the `os.Stdin` through a parameter?

Reply:
the parameter design to test simple.